src/BaseDAO/ShareTest/Token.hs

```diff
@@ -0,0 +1,114 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+module BaseDAO.ShareTest.Token
+  ( adminBurnScenario
+  , adminMintScenario
```

Comment:
I think we should remove `admin` from the names here, since they also check for non-admin usage